src/server/redis_reply.cc

```diff
@@ -37,9 +37,23 @@ namespace redis {
 
 void Reply(evbuffer *output, const std::string &data) { evbuffer_add(output, data.c_str(), data.length()); }
 
-std::string SimpleString(const std::string &data) { return "+" + data + CRLF; }
+std::string SimpleString(const std::string &data) { 
```

Comment:
Can the input also be a `std::string_view`?

Reply:
Of course! In fact, I would really like to switch to using std::string_view as the parameter type. However, I saw that const string& is widely used in the code, so I assumed it was a stylistic choice and didn't make the change. If using the former is acceptable, I’ll make the adjustment later.

Reply:
> However, I saw that const string& is widely used in the code, so I assumed it was a stylistic choice and didn't make the change.

Previously `+` for string_view would be a bit trick, but now there is no reason not shift to that. For sso string the performance might be same but for long string `string_view` would have better performance

Reply:
For some cases (e.g. we need to pass a null-terminated string to a C api), `std::string` is required since `std::string_view` will not ensure that the string is terminated by `\0`.
In other cases, it can be replaced by std::string_view if the lifetime can be maintained well (no dangling happen).

Reply:
However, previously I was considering to introduce "rope" (a data structure) to build `concat`ed strings in a relatively cheaper way, but it may require many non-trivial work.

Reply:
> However, previously I was considering to introduce "rope" (a data structure) to build `concat`ed strings in a relatively cheaper way, but it may require many non-trivial work.

It's ok for some network buffer but would be to complex for `SimpleString`...